README.md

```diff
@@ -47,16 +47,16 @@ php -d memory_limit=-1 composer.phar create-project --no-dev ezsystems/ezplatfor
 Then, create a new database using the following command:
 
 ```bash
-php bin/console doctrine:database:create
+php app/console doctrine:database:create
```

Comment:
you seems to be installing 1.x and not 2.x or?

Reply:
v2 is using symfony 3.x, and hence need bin/console.

If you are looking for install instruction on 1.x, go to 1.13 branch. Or ideally online doc which should be the central place to go to.